keras_hub/src/utils/tensor_utils.py

```diff
@@ -322,3 +322,107 @@ def any_equal(inputs, values, padding_mask):
         output = ops.logical_or(output, value_equality)
 
     return ops.logical_and(output, padding_mask)
+
+
+def target_gather(
+    targets,
+    indices,
+    mask=None,
+    mask_val=0.0,
+):
+    """A utility function wrapping tf.gather, which deals with:
```

Comment:
this utility is not really wrapping `tf.gather` but it is reimplementing right? lets update the doc-string to remove tf.gather.